samples/vboxwrapper/vbox_mscom61.cpp

```diff
@@ -0,0 +1,64 @@
+// This file is part of BOINC.
+// http://boinc.berkeley.edu
+// Copyright (C) 2010-2012 University of California
+//
+// BOINC is free software; you can redistribute it and/or modify it
+// under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation,
+// either version 3 of the License, or (at your option) any later version.
+//
+// BOINC is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+// See the GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with BOINC.  If not, see <http://www.gnu.org/licenses/>.
+
+#define _VIRTUALBOX60_
```

Comment:
Shouldn't this be
`#define _VIRTUALBOX61_`

Reply:
No. Maybe it's a good idea to rename `_VIRTUALBOX60_` to `_VIRTUALBOX6_` and change its usage